pkgs.nix

```diff
@@ -4,7 +4,8 @@ let
   lib = pkgs.lib;
   withPlugins = pluginsFunc:
     let plugins = pluginsFunc availablePlugins;
-    in pkgs.python3Packages.buildPythonApplication {
+    in
```

Comment:
I’m not sure if there’s a standard in nixpkgs between nixfmt and nixpkgs-fmt, but I’m using nix-mode for Emacs which uses nixfmt, so I would prefer to keep the formatting style of nixfmt. ;)

Reply:
That's my LSP rnix that chooses this mode of formatting I'll change it back.